dolphinscheduler-api/src/main/java/org/apache/dolphinscheduler/api/service/impl/TaskInstanceServiceImpl.java

```diff
@@ -258,6 +258,7 @@ public Result forceTaskSuccess(User loginUser, long projectCode, Integer taskIns
 
         // change the state of the task instance
         task.setState(TaskExecutionStatus.FORCED_SUCCESS);
+        task.setEndTime(new Date());
```

Comment:
Could u plz add a corresponding unit test case to cover this patch? Thanks

Reply:
> Could u plz add a corresponding unit test case to cover this patch? Thanks


Forced success only results in a lack of end time, which prevents the runtime from terminating,I don't think this unit test is necessary，Thanks